Prace_domowe/Praca_domowa2/Grupa4/Marcin Wilk/PD2.ipynb

```diff
@@ -0,0 +1,1617 @@
+{
```

Comment:
<p>Ja nie umiem wymówić "regularyzowaną różnicę", a Ty? </p><p>Co więcej ,nie wiem co to znaczy, (choć się domyślam). </p><p>Przetłumaczenie dokumentacji to mało ambitne podejście, zwłaszcza na tak elytarnym kierunku jakim jest IAD</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>